pkg/gen/input/workflows/internal/file/validate_changelog.yaml.template

```diff
@@ -0,0 +1,34 @@
+{{{{ .Header }}}}
+name: Validate Release Changelog
+
+on:
+  workflow_call:
+  pull_request:
+    types: [opened, synchronize, reopened]
+    paths:
+      - 'CHANGELOG.md'
+    branches:
+      - 'legacy#release#v*.*.*'
+      - 'main#release#v*.*.*'
+      - 'main#release#major'
+      - 'main#release#minor'
+      - 'main#release#patch'
+      - 'master#release#v*.*.*'
+      - 'master#release#major'
+      - 'master#release#minor'
+      - 'master#release#patch'
+      - 'release#v*.*.*'
+      - 'release#major'
+      - 'release#minor'
+      - 'release#patch'
+      - 'release-v*.*.x#release#v*.*.*'
+      # "!" negates previous positive patterns so it has to be at the end.
+      - '!release-v*.x.x#release#v*.*.*'
+
+permissions:
+  contents: read
+  pull-requests: write
+
+jobs:
+  validate-changelog:
+    uses: giantswarm/github-workflows/.github/workflows/validate-changelog.yaml@validate-changelog
```

Comment:
Will be updated once https://github.com/giantswarm/github-workflows/pull/28 is merged

Reply:
Thanks for placing the actual workflow in the right place (giantswarm/github-workflows)!